packages/features/bookings/lib/get-booking.ts

```diff
@@ -161,3 +167,64 @@ export const getBookingByUidOrRescheduleUid = async (uid: string) => {
       : booking.attendees,
   };
 };
+
+/**
+ * Should only get booking attendees length for seated events
+ * @param uid
+ * @returns booking with masked attendee emails
+ */
+export const getBookingForSeatedEvent = async (uid: string) => {
```

Comment:
New function that should fetch booking considering data return for booking with seats